src/components/card/card.scss

```diff
@@ -0,0 +1,196 @@
+
+@import "variables";
+@import "shadows";
+@import "default-theme";   // TODO: Remove this
+
+$md-card-default-padding: rem(2.400) !default;
+$md-card-mobile-padding: rem(2.400) rem(1.600) !default;
+$md-card-border-radius: 2px !default;
+$md-card-header-size: rem(4.000) !default;
```

Comment:
Can you convert the `rem` values to what they actually evaluate to? The use of `rem` is just a leftover from material 1 that should be undone.